main.nf

```diff
@@ -638,23 +649,47 @@ process multiqc {
     !params.skip_qc && !params.skip_multiqc
```

Comment:
You will need to update these parameter names too. Check all of the parameters for the pipeline are the same in `nextflow.config`, `main.nf`, `usage.md` and `output.md`.

Reply:
bad Lorena, bad Lorena....I'll do that, thanks for catching it.

Reply:
I tried to make sure all is the same now.